src/tools/crossgen2/Common/TypeSystem/Common/MethodDesc.cs

```diff
@@ -24,6 +24,13 @@ public enum MethodSignatureFlags
         Static = 0x0010,
     }
 
+    public struct CustomModifier
```

Comment:
Rename this type to indicate it can be used to hold other obscure signature comparison effecting logic as well (such as the lo and hi bounds of the ELEMENT_TYPE_ARRAY). Something like EmbeddedSignatureData, and instead of bool required, use an enum.